broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -440,6 +440,11 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
             return response;
         }
 
+        if (topicConfig.equals(this.brokerController.getTopicConfigManager().getTopicConfigTable().get(topic))) {
+            response.setCode(ResponseCode.SUCCESS);
+            return response;
+        }
```

Comment:
It would be better to add a log indicating that the topic already exists and will no longer be created or updated repeatedly.